### PR TITLE
8.0 Port account chart update

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

File: account_chart_update/wizard/wizard_chart_update.py [hunk 1]
```diff
@@ -272,7 +272,8 @@ def _get_code_digits(self, cr, uid, context=None, company_id=None):
         if property_ids:
             prop = property_obj.browse(
                 cr, uid, property_ids[0], context=context)
-            account_id = prop.value_reference.id
+            account_id = len(prop.value_reference.split(',')) == 2 and\
```
Comment (guewen) [inline account_chart_update/wizard/wizard_chart_update.py]: `prop.value_reference` can be False so `prop.value_reference.split(',')` will fail

Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py]: Maybe this should be `account_id = prop.value_reference.res_id`?

**Comment by pedrobaeza** [comment]
Thanks for the changes. At first sight, it seems correct to me :+1: (code review)

Next week, I will make functional test and also feed this PR.

**Comment by adrienpeiffer** [comment]
Thanks @pedrobaeza !

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**